docs/data/base/getting-started/overview/overview.md

```diff
@@ -1,22 +1,23 @@
 # MUI Base - Overview
 
-<p class="description">MUI Base is a library of blank ("unstyled") React UI components and low-level hooks.</p>
+<p class="description">MUI Base is a library of headless ("unstyled") React UI components and low-level hooks.</p>
 
 ## Introduction
 
 MUI Base is a library of unstyled React UI components.
-It features the same robust engineering as [Material UI](/material-ui/getting-started/overview/), but without implementing Material Design.
+The components were extracted from [Material UI](/material-ui/getting-started/overview/).
+It features the same robust engineering but without implementing Material Design.
 
 It includes prebuilt components with production-ready functionality, along with low-level hooks for transferring that functionality to other components.
 
 With MUI Base, you can rapidly build on top of our foundational components using any styling solution you choose—no need to override any default style engine or theme.
 
 :::info
-**Note:** _as of Q2 2022, Base is currently still in alpha._
+**Note:** as of today, MUI Base is in alpha.
 
-We're adding new components regularly, and you're welcome to contribute!
+We're adding new components and features regularly, and you're welcome to contribute!
 
-Look for the [`package: base` label](https://github.com/mui/material-ui/labels/package%3A%20base) on open [issues](https://github.com/mui/material-ui/issues) and [pull requests](https://github.com/mui/material-ui/pulls) in the `@mui/material-ui` GitHub repository to see what other community members are working on, and feel free to submit your own.
```

Comment:
`@mui/material-ui` only make sense if referring to the npm package, I don't think that in the context of a GitHub repository developer would understand it.

Reply:
Leaving "label" outside of the link makes it easier to navigate:

<img width="740" alt="Screenshot 2022-05-31 at 01 48 43" src="https://user-images.githubusercontent.com/3165635/171068901-10534706-e1a5-4e11-bb9d-a419ffbc2535.png">